hack/deploy.sh

```diff
@@ -39,11 +39,11 @@ echo "Using project ${PROJECT_ID} and user ${PROJECT_USER}"
 gcloud config set project ${PROJECT_ID}
 
 existing_cluster=$(gcloud container clusters list \
-    --zone=${K8S_CLUSTER_ZONE} --filter="name=${K8S_CLUSTER_NAME}")
+    --zone=${K8S_CLUSTER_REGION}-${K8S_CLUSTER_ZONE} --filter="name=${K8S_CLUSTER_NAME}")
```

Comment:
was the script broken without region input?

Reply:
Yes.